command/snapshot/inspect/snapshot_inspect.go

```diff
@@ -101,7 +114,7 @@ func (c *cmd) Run(args []string) int {
 		}
 	}()
 
-	stats, totalSize, err := enhance(readFile)
+	stats, statsKV, totalSize, totalSizeKV, err := enhance(readFile, c.detailed, c.depth, c.filter)
```

Comment:
Discussed having flag check to do something like the following logic:
If Detailed (name tbd) is not set but the other flags are called, set detailed to true?